torch/__init__.py

```diff
@@ -616,7 +624,7 @@ def compiled_with_cxx11_abi():
 from torch._classes import classes
```

Comment:
This is also unused; maybe worthwhile checking each of these to see if there are any unused.